internal/hud/hud.go

```diff
@@ -161,8 +161,10 @@ func (h *Hud) handleScreenEvent(ctx context.Context, dispatch func(action store.
 			h.selectedScroller(h.r.rty).Bottom()
 		case tcell.KeyEnter:
 			if !h.currentViewState.LogModal.IsActive() {
-				selectedIdx, _ := h.selectedResource()
-				h.currentViewState.LogModal = view.LogModal{ResourceLogNumber: selectedIdx + 1}
+				selectedIdx, r := h.selectedResource()
+				if !r.IsYAMLManifest {
+					h.currentViewState.LogModal = view.LogModal{ResourceLogNumber: selectedIdx + 1}
+				}
```

Comment:
I think we want to log an error here, so the user doesn't just think it's a bug.

(and then we'll turn it into a proper error with [ch702](https://app.clubhouse.io/windmill/story/702/hud-error-messages-should-appear-in-the-hud))

Reply:
yep!